<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buyers</class>
 <widget class="QDialog" name="Buyers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj kontrahenta</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>Nazwa*:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel3">
         <property name="text">
          <string>Miejscowość*:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4">
         <property name="text">
          <string>Kod*:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel2">
         <property name="text">
          <string>Adres*:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="textLabel2_2">
           <property name="text">
            <string>NIP*:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="gusBtn">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Noto Sans</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GUS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5">
         <property name="text">
          <string>Nr konta:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel6">
         <property name="text">
          <string>Typ*:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="placeEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="codeEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="addressEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="nipEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>13</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="accountEdit">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo">
         <item>
          <property name="text">
           <string>Firma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Urząd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Osoba fizyczna</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel5_2">
         <property name="text">
          <string>Telefon:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2_2">
         <property name="text">
          <string>e-mail</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2_2_2">
         <property name="text">
          <string>www:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLineEdit" name="telefonEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="emailEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="wwwEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="gusLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>O&amp;K</string>
       </property>
       <property name="shortcut">
        <string>Alt+K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Anuluj</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Buyers</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
